ntbs-service/DataAccess/UserRepository.cs

```diff
@@ -12,6 +12,7 @@ public interface IUserRepository
         Task AddOrUpdateUser(User user, IEnumerable<TBService> tbServices);
         Task AddUserLoginEvent(UserLoginEvent userLoginEvent);
         Task<User> GetUserByUsername(string username);
+        IQueryable<User> GetUserIQueryable();

```

Comment:
Very minor point, but I think `GetUserQueryable` is a better name than `GetUserIQueryable` - it's just as descriptive and easier to read.